import.go

```diff
@@ -7,97 +7,89 @@ import (
 	"io"
 	"os"
 	"regexp"
+	"text/template"
 
 	errors "github.com/charter-se/structured/errors"
 	"github.com/spf13/viper"
 )
 
-type TemplateFunc func([]byte) ([]byte, error)
-
-//YamlFile stores raw file bytes and Viper struct
-type YamlSection struct {
-	Bytes []byte
-	Viper *viper.Viper
-}
-
-//Import reads data from a single YAML file and adds its data to this *Yp instance
-func (yp *Yp) ImportFile(s string) error {
-	r, err := os.Open(s)
-	if err != nil {
-		return err
-	}
-	yf, err := importYaml(bufio.NewReader(r))
+//Import takes a location identifier (URI, file path, etc..) and an io.Reader
+//imported data is added to the yamlPack instance
+func (yp *Yp) Import(s string, r io.Reader) error {
```

Comment:
Would like to see these string arguments named for what they are, makes it a little easier to understand the code without needing to trackback to this function, also easier for newer devs to understand.